yarn-project/noir-protocol-circuits/src/crates/types/src/abis/private_circuit_public_inputs.nr

```diff
@@ -79,3 +79,68 @@ impl PrivateCircuitPublicInputs{
         dep::std::hash::pedersen_hash_with_separator(fields.storage, constants_gen::GENERATOR_INDEX__PRIVATE_CIRCUIT_PUBLIC_INPUTS)
     }
 }
+
+struct PrivateCircuitPublicInputsBuilder {
```

Comment:
Maybe we can move this to the test folder together with the public circuit public inputs builder?

Reply:
Hahaha glad you think so cause I already did it in the refactoring PR (coming soon)!